src/domain/Loan.ts

```diff
@@ -16,16 +16,24 @@ export function updateLoans(pool: Pool, pileAddress: string): BigInt[] {
     let loans = pool.loans
     let loanId = loans[j]
 
+    let loan = Loan.load(loanId)
+    if (!loan) {
+      log.error('updateLoans: loan {} not found', [loanId])
+      return [new BigInt(0), new BigInt(0)]
```

Comment:
I actually think this should also be
```suggestion
      continue
```